userspace/falco/falco.cpp

```diff
@@ -86,6 +86,7 @@ static void usage()
 	   " -h, --help                    Print this page\n"
 	   " -c                            Configuration file (default " FALCO_SOURCE_CONF_FILE ", " FALCO_INSTALL_CONF_FILE ")\n"
 	   " -A                            Monitor all events, including those with EF_DROP_SIMPLE_CONS flag.\n"
+	   " --alternate-lua-dir <path>    Specify an alternate path for loading falco lua files\n"
```

Comment:
This is probably better as `--lua-dir <path>` and in the description we can add `default " FALCO_ENGINE_SOURCE_LUA_DIR"`

Reply:
it is not actually the lua dir. The main lua dir will still be specified at compile time from installation path. Is `--fallback-lua-dir` better?

Reply:
I actually took a closer look at the code, it looks like t he two variables are separate internally so `--alternate-lua-dir` is a good option to me.

Reply:
I vote for `--alternate-lua-dir`